docs/android/v2/plugins/Virtual-background.mdx

```diff
@@ -0,0 +1,177 @@
+---
+title: Virtual Background
+nav: 12
+---
+
+## Introduction
+
+Virtual background plugin helps in customising one’s background i.e. replacing the background with a static image.
+This guide provides an overview of usage of the virtual background plugin of 100ms.
+
+![VirtualBackground](/docs/v2/vb.jpg)
+
+## Supported Android API Levels
+
+Virtual background plugin supports Android API level 21 and higher and minimum sdk version support is 21. It is built for armeabi-v7a, arm64-v8a, x86, and x86_64 architectures.
```

Comment:
```suggestion
Min api version for the virtual background plugin is 21. Same as the min api version core sdk for 100ms. The lowest SDK version it can work with is REPLACE_THIS. It is built for armeabi-v7a, arm64-v8a, x86, and x86_64 architectures.
```